db/migrate/20171208063317_add_index_to_users_email.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToUsersEmail < ActiveRecord::Migration[5.1]
```

Comment:
tại sao phải tách riêng ra 2 file?

Reply:
add index cho email của user sẽ tiện cho việc truy vấn dữ liệu, thay vì truy vấn toàn bộ thuộc tính của bảng user ta sẽ tìm đến id của user đó

Reply:
tách riêng để dễ dàng cho việc truy vấn dữ liệu thay vì truy vấn tất cả các thuộc tính của bảng user để lấy dữ liệu cần tìm thì ta tìm theo id email của user đó